xcodeproj/internal/target.bzl

```diff
@@ -262,13 +227,12 @@ def _processed_target(
     return struct(
         defines = defines,
         dependencies = dependencies,
-        extra_files = extra_files,
-        generated = generated + [
-            file
-            for file in modulemaps.files
-            if not file.is_source
-        ],
-        inputs_info = inputs_info,
+        # generated = generated + [
+        #     file
+        #     for file in modulemaps.files
+        #     if not file.is_source
+        # ],
```

Comment:
I need to address this till. Forgot about it.

It's interesting that the specs haven't changed, which means they are being found elsewhere, but maybe not reliably.